spack/packages/fiber/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from spack import *
+
+FFT_LIBS = (
+    'heffte',
+    'fftw',
+    'ffte',
+)
+
+
+class Fiber(CMakePackage):
+    """High Performance Fast Fourier Transform benchmark."""
+
+    homepage = "https://fiber.icl.utk.edu/"
```

Comment:
I think alan has just not changed the site since I set it up for him with the heffte template.  There is a separate heffte.icl.utk.edu for the heffte profiler.  Anyway, it is not a critical part of the package at this point.